terragrunt/env/production/cloudfront/terragrunt.hcl

```diff
@@ -0,0 +1,49 @@
+terraform {
+  source="git::https://github.com/cds-snc/url-shortener//terragrunt/aws/cloudfront?ref=${get_env("INFRASTRUCTURE_VERSION")}"
+}
+
+dependencies {
+  paths = ["../hosted_zone", "../network", "../api"]
+}
+
+dependency "hosted_zone" {
+  config_path = "../hosted_zone"
+
+  mock_outputs_allowed_terraform_commands = ["init", "fmt", "validate", "plan", "show"]
+  mock_outputs_merge_with_state           = true
+  mock_outputs = {
+    hosted_zone_id = "1234567890"
+  }
+}
+
+dependency "network" {
+  config_path = "../network"
+
+  mock_outputs_allowed_terraform_commands = ["init", "fmt", "validate", "plan", "show"]
+  mock_outputs_merge_with_state           = true
+  mock_outputs = {
+    vpc_id = "url_shortener_vpc_id"
+  }
+}
```

Comment:
Do we need the `vpc_id` for the CloudFront module?  I didn't see it, but I could also have completely missed it 😅

Reply:
So I am using the vpc id here but I might not need it?:
```
#
# Route53 DNS logging and query firewall
#
module "resolver_dns" {
  source           = "github.com/cds-snc/terraform-modules?ref=v5.0.2//resolver_dns"
  vpc_id           = var.vpc_id
  firewall_enabled = true

  allowed_domains = [
    "*.amazonaws.com.",
    "*.gc.ca.",
    "*.canada.ca.",
  ]

  billing_tag_value = var.billing_code
}
```